mmdeploy/codebase/mmdet/deploy/object_detection_model.py

```diff
@@ -255,7 +256,13 @@ def forward(self,
                 # avoid to resize masks with zero dim
                 if rescale and masks.shape[0] != 0:
                     masks = torch.nn.functional.interpolate(
-                        masks.unsqueeze(0), size=(ori_h, ori_w))
+                        masks.unsqueeze(0),
+                        size=[
+                            math.ceil(masks.shape[-2] /
+                                      img_metas[i]['scale_factor'][0]),
```

Comment:
Is it possible that scale_factor here is a Int value?

Reply:
Another risk from mmyolo transformation  is `letter_resize`.

Reply:
The type of scale_factor is float according to mmcv's definition.

Reply:
I mean img_metas[i]['scale_factor'] is `float` instead of `tuple[float]`

Reply:
The type is Tuple[float]. It is the convention in mmcv.